crates/holochain/src/conductor/conductor.rs

```diff
@@ -1405,6 +1405,7 @@ mod app_impls {
             };
 
             for (dna, _) in ops.dnas_to_register {
+                self.spaces.get_or_create_space(dna.dna_hash())?;
```

Comment:
```suggestion
```

We can delete this as it's already done by [genesising the cell](https://github.com/holochain/holochain/blob/develop/crates/holochain/src/conductor/conductor.rs#L1417), which in turn [creates the space](https://github.com/holochain/holochain/blob/develop/crates/holochain/src/conductor/conductor.rs#L2918).